### DISCUSSION TITLE
How to obtain a average field when I calculate a unsteady case, such as turb_ONERAM6

### DISCUSSION THREAD

**Discussion by crazyfireji** [su2code/SU2#2319] [Q&A]
As I mentioned in title, I want to calculate the time average flow field(u_ave, v_ave, p_aver...) when I run an unsteady case（v8.0.1）. I dont know wheter WINDOW_START_ITER can achieve it, and how to control the name of average flow fields.

**Comment by pcarruscag** [comment]
We do compute the time average of those variables for the entire volume. You can use the dry run mode to list the available variables, check the documentation on how to control outputs.

**Reply by bigfooted** [reply]
Look for the keyword `TIME_AVERAGE` which outputs all available time averages. With `WINDOW_START_ITER ` you tell SU2 when to start averaging.
Btw, I noticed that we only do time averaging of density,pressure and velocities+rms. So no temperature afaik. 
@pcarruscag Can we construct time averaging with the macros?
